src/org/skypro/skyshop/search/BestResultNotFound.java

```diff
@@ -0,0 +1,7 @@
+package org.skypro.skyshop.search;
+
+public class BestResultNotFound extends Exception {
```

Comment:
Исключения лучше называть с окончанием на Exception
```suggestion
public class BestResultNotFoundException extends Exception {
```